Resources/public/js/components/automation-tab/main.js

```diff
@@ -263,11 +268,23 @@ define([
                 );
 
                 if (!data.id) {
-                    this.updateNotification(++this.notificationBadge);
+                    this.updateNotification(this.incrementNotificationBadge());
                 }
             }.bind(this));
         },
 
+        incrementNotificationBadge: function() {
+            return ++this.notificationBadge;
+        },
+
+        decrementNotificationBadge: function() {
+            if (this.notificationBadge === 0) {
```

Comment:
Isn't that the exact fix for the problem you've described? I would only change it to `this.notificaitonBadge <= 0`, would be safer.